config/manager/manager.yaml

```diff
@@ -3,7 +3,7 @@ kind: Namespace
 metadata:
   labels:
     control-plane: controller-manager
-  name: system
+  name: harbor-cluster-operator-system
```

Comment:
It will be override by `config/default/kustomization.yaml`. Do not update.

Reply:
ok, why I changed it because the namespace is not overridden in my env. It seems to be a bug of kustomize.